cavert-pkirch/hello/hello.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import "fmt"
+
+const spanish = "Spanish"
+const french = "French"
+const englishHelloPrefix = "Hello, "
+const spanishHelloPrefix = "Hola, "
+const frenchHelloPrefix = "Bonjour, "
+const helloPostfix = "!"
+
+func Hello(name string, language string) string {
```

Comment:
Public functions (with uppercase letters at the beginning) will be exported and should have a documentation in the form: 

```suggestion
// Hello takes in a name and language and returns a greeting string
func Hello(name string, language string) string {
```